tdrs-backend/tdpservice/parsers/schema_defs/tanf/t3.py

```diff
@@ -80,12 +81,12 @@
               ]),
         Field(item="68", name='DATE_OF_BIRTH', type='number', startIndex=20, endIndex=28,
               required=True, validators=[
-                  validators.month_year_yearIsLargerThan(1998),
-                  validators.month_year_monthIsValid(),
+                  validators.dateYearIsLargerThan(1998),
+                  validators.dateMonthIsValid(),
               ]),
-        EncryptedField(decryption_func=tanf_ssn_decryption_func, item="69", name='SSN', type='string', startIndex=28,
-                       endIndex=37, required=True, validators=[validators.notOneOf([str(i)*9 for i in range(0, 10)])
-                                                               ]),
+        TransformField(transform_func=tanf_ssn_decryption_func, item="69", name='SSN', type='string', startIndex=28,
+                       endIndex=37, required=True, validators=[validators.notOneOf([str(i)*9 for i in range(0, 10)])],
```

Comment:
```suggestion
                       endIndex=37, required=True, validators=[validators.validateSSN()],
```

Did we miss converting to this new function you made?

Reply:
Just hadnt merged develop yet.